pkg/pipeline/pipeline.go

```diff
@@ -261,7 +270,10 @@ func (p *Pipeline) Run(ctx context.Context) *livekit.EgressInfo {
 			manifestLocalPath := fmt.Sprintf("%s.json", p.PlaylistFilename)
 			manifestStoragePath := fmt.Sprintf("%s.json", playlistStoragePath)
 			if err := p.storeManifest(ctx, manifestLocalPath, manifestStoragePath); err != nil {
-				logger.Errorw("could not store manifest", err)
+				logger.Infow("could not store manifest", "error", err)
```

Comment:
this is currently double logging (could not upload file + could not store manifest)